omod/src/main/java/org/openmrs/module/locationbasedaccess/fragment/controller/field/LocationsFragmentController.java

```diff
@@ -0,0 +1,36 @@
+package org.openmrs.module.locationbasedaccess.fragment.controller.field;
+
+import org.openmrs.Location;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.appui.UiSessionContext;
+import org.openmrs.ui.framework.fragment.FragmentModel;
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Fields and actions to provide registrationInfo view.
+ *
+ */
+public class LocationsFragmentController {
+
+    public void controller(FragmentModel model,
+                           HttpSession session) {
+        String LOCATION_SESSION_ATTRIBUTE = UiSessionContext.LOCATION_SESSION_ATTRIBUTE;
```

Comment:
Did you look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Changed